coverage.sh

```diff
@@ -1,12 +1,5 @@
 #!/usr/bin/env bash
 
 set -e
-echo "" > coverage.txt
-
-for d in $(go list ./... | grep -v vendor); do
-    go test -race -coverprofile=profile.out -covermode=atomic $d
-    if [ -f profile.out ]; then
-        cat profile.out >> coverage.txt
-        rm profile.out
-    fi
-done
+go list ./... | grep -v vendor | parallel -k go test -race -coverprofile=out/{#} -covermode=atomic {}
```

Comment:
I think you may find that `xargs` with the help of appropriate `-P` and `-n` flags might do the trick, and be more portable?